test/sslapitest.c

```diff
@@ -4948,10 +4948,14 @@ static int test_key_exchange(int idx)
             break;
 #  ifndef OPENSSL_NO_ECX
         case 4:
+            if (is_fips)
+                return TEST_skip("X25519 might not be supported by fips provider.");
```

Comment:
well "X25519 should not be supported by fips provider" is more accurate, as any instances that did claim such support are unapproved back then through to present.

Reply:
Currently should not, maybe in future it could be. I think this is ok as is.